src/callback.jl

```diff
@@ -39,7 +39,7 @@ function _pyjlwrap_call(f, args_::PyPtr, kw_::PyPtr)
 
         return pyreturn(ret)
     catch e
-        pyraise(e)
+        pyraise(e, catch_backtrace())
```

Comment:
Might be cleaner to have a `@pyraise e` macro that inserted the `catch_backtrace()` call:
```jl
macro pyraise(e)
    :(pyraise($(esc(e)), catch_backtrace())
end
```

Reply:
Done